File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { FreeGamesSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await FreeGamesSDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- FreeGames SDK exists test

local sdk = require("free-games_sdk")

describe("FreeGamesSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
